bootstrap-salt.sh

```diff
@@ -3032,10 +3032,10 @@ __install_saltstack_rhel_repository() {
         gpg_key="SALTSTACK-GPG-KEY.pub"
     fi
 
-    repo_file="/etc/yum.repos.d/salt-${repo_rev}.repo"
+    repo_file="/etc/yum.repos.d/saltstack.repo"
```

Comment:
I like this change.  We should probably change the repo name on ubuntu/debian and others to `saltstack` since it is likely that you'd want only one saltstack repo configured at a time.